ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -461,6 +464,10 @@ public void replicate() {
     } finally {
       replicationMetrics.updateOneCycleReplicationTime(time.milliseconds() - oneRoundStartTimeMs,
           replicatingFromRemoteColo, datacenterName);
+      groupFinishTime.forEach((groupId, finishTime) -> {
```

Comment:
Ms granularity is too fine for this metric. We don't care at that granularity. There is not much we can do and not worth optimizing for milliseconds. We care if a group idle for a minute or more or even many seconds. Since we don't know that yet, I'd say measure this in seconds.

Reply:
Added in Ms, while plotting we can change in Seconds. Seeing fractional values are not supported in our Histogram.